tutorials/tts/Aligner_Inference_Examples.ipynb

```diff
@@ -0,0 +1,746 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2d53c8e4",
+   "metadata": {},
+   "source": [
+    "We'll need to import some libraries for loading audio, plotting various data, and of course for loading the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1d463d1e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start with some imports so we can visualize alignments and load the checkpoint\n",
+    "%matplotlib inline\n",
+    "import matplotlib.pylab as plt\n",
+    "import IPython.display as ipd\n",
+    "\n",
+    "import librosa\n",
+    "import soundfile as sf\n",
+    "import torch\n",
+    "\n",
+    "from nemo.collections.tts.models import AlignerModel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d30138d",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "Let's start by loading the checkpoint, which should have automatically been saved by the trainer as a file in the output directory's `checkpoints/` folder called `Aligner.nemo`.\n",
+    "\n",
+    "Please update `ckpt_path` in the cell below to the appropriate location on your machine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "896fee82",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set device (GPU or CPU)\n",
+    "device = torch.device(\"cuda:0\" if torch.cuda.is_available() else \"cpu\")\n",
+    "\n",
+    "# Replace the following with the path to your Aligner model checkpoint\n",
+    "ckpt_path = \"<your_checkpoint_path_here>/Aligner.nemo\"\n",
+    "aligner = AlignerModel.restore_from(ckpt_path, map_location=device)\n",
+    "\n",
+    "# This should be set to whatever sample rate your model was trained on\n",
+    "target_sr = 22050"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c62d2786",
+   "metadata": {},
+   "source": [
+    "Now we'll load an audio file and input the corresponding transcript. The audio file will be resampled to the `target_sr` given above.\n",
+    "\n",
+    "This example uses the first sample from the NVIDIA test split of LJSpeech, which is file `LJ023-0089.wav`. You can use whatever you'd like, of course, but this tutorial will refer to this sample specifically for a concrete example.\n",
+    "\n",
+    "If you'd like to download the LJSpeech dataset to use the same example, you can find it [here](https://keithito.com/LJ-Speech-Dataset/).\n",
+    "\n",
+    "Please update `audio_path` to the sample you'd like to use, and `text_raw` if it differs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7f3bbaf7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This tutorial uses a sample from the NVIDIA test split of LJSpeech.\n",
+    "# Please change the following two lines to reflect your audio file and transcript.\n",
+    "audio_path = \"<your_data_path>/LJ023-0089.wav\"\n",
```

Comment:
how about putting this example on AWS?

Reply:
Discussed over Slack--updated. Thanks for uploading!